### PR TITLE
feat: SDK-232 unified TC

### PR DESCRIPTION
**Pull request by wackyleo459**
- Cycles Balance on top will reflect in TC. When balance falls below 1TC, balance to the decimal will be shown.

- Exact amount of cycles is viewable as popover, when hovering over balance in Cycles Balance.

- Created canisters reflect cycles in TC.

- Modal to create new canister shows cycles in both C(cycles) and TC.

- Warning error appears if allotting less than 2TC when creating canister.

### CODE REVIEW

File: wallet_ui/components/routes/Dashboard.tsx [hunk 1]
```diff
@@ -134,16 +152,15 @@ export function Dashboard(props: { open: boolean; onOpenToggle: () => void }) {
...
-  };
-
   function handleWalletCreateDialogClose(maybeErr?: any) {
     setWalletCreateDialogOpen(false);
     setErrorDialogContent(maybeErr);
   }
 
+  renderInstance.current++;
+
+  console.log("render Dashboard", renderInstance);
```
Comment (krpeacock) [inline wallet_ui/components/routes/Dashboard.tsx:162]: extra console.log

File: wallet_ui/utils/cycles.ts [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
-export function format_cycles_and_suffix(cycles : bigint) {
+export function format_cycles_trillion_fullrange(cycles: bigint) {
   const cycles_float = parseFloat(cycles.toString());
-  const SUFFIX_LIST = " KMGTPE";
-  const suffix = SUFFIX_LIST[Math.min(6, Math.floor(Math.log10(cycles_float) / 3))];
-  const ll = SUFFIX_LIST.indexOf(suffix);
-  const humanCycles = Math.floor(cycles_float / 10 ** (ll * 3));
-  return [humanCycles.toString(), suffix];
-}  
+  const units = {
```
Comment (krpeacock) [inline wallet_ui/utils/cycles.ts:13]: Are you still using these units? I recall you saying you were going to go with TC across the board

**Review by krpeacock — CHANGES_REQUESTED** [review]
a couple small changes, but it otherwise looks good.

This is still too large of a PR though. There are at least four smaller changes here that would be easier to review separately

- formatting changes + eslint
- minimum cycles in slider
- managed events and custom names
- TC units

That adds a lot of overhead and makes it harder for me to get my head around what is a meaningful change for the current ticket. Otherwise I have to work backwards to figure out what is changing in a bunch of areas and why

File: wallet_ui/utils/cycles.ts (hunk 2 above)
Comment (wackyleo459) [reply wallet_ui/utils/cycles.ts:13]: No I'm not using those units anymore. They are replaced with different standard of decimals.

**Comment by wackyleo459** [comment]
I've made requested changes. Eslint packages were removed, as it was more for personal use case. Next time, I will isolate to fewer relevant changes per ticket!

**Review by krpeacock — APPROVED** [review]

**Pull request merged**